### PR TITLE
Refactoring the code

### PR DESCRIPTION
**Pull request by arianaosorio**

### CODE REVIEW

File: life_expectancy/cleaning.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+"""
+Module that contains all functions to clean dataset.
```
Comment (danizao) [inline life_expectancy/cleaning.py:2]: Good comment here, explaining what the module is doing!

File: life_expectancy/cleaning.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    dataset = pd.melt(
+        dataset, id_vars=["unit", "sex", "age", "region"], var_name="year"
+    )
+
+    dataset["year"] = dataset["year"].astype("int")
+    dataset["value"] = dataset.value.str.extract(r"(\d+\.\d)").astype("float")
+
+    clean_dataset = dataset.dropna()
+
+    dataset_output = clean_dataset[clean_dataset.region == country]
```
Comment (danizao) [inline life_expectancy/cleaning.py:63]: Do you really ned to create other variables here (clean_dataset and dataset_output)?

File: life_expectancy/cleaning.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+def clean_data(dataset: pd.DataFrame, country: str = "PT") -> pd.DataFrame:
+    """
+    Clean the raw data.
+
+    :param dataset: A pandas dataframe with the data to clean.
+    :param country: The name of country.
+    :returns: A pandas dataframe with the cleaned data.
+    """
+    dataset[["unit", "sex", "age", "region"]] = dataset[
```
Comment (danizao) [inline life_expectancy/cleaning.py]: Usually i have been adviced to start with a copy of the dataset so we don't mess with the main dataset that is being passed:
data = dataset.copy()

**Review by danizao — COMMENTED** [review]
In resume:
- good comments in docstring format (well done!)
- No magic numbers - just because Fernando asked to check this :-)
- variable names look descriptive
- some variables looked duplicated (file_path)

File: life_expectancy/cleaning.py (hunk 2 above)
Comment (arianaosorio) [reply life_expectancy/cleaning.py:63]: in my opinion, it's more clear if I define those variables. Then when I read the code i know that function clean dataset and return a dataset for a specific region. @MrCordeiro what is your opinion about that? :)

**Pull request closed without merging**